### PR TITLE
Automatically push tagged commits to rubygems

### PR DESCRIPTION
**Pull request by jcarres-mdsol**
@mdsol/team-10

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
+- 2.3.0
+- 2.4.2
 script: bundle exec rspec
-
 before_install:
-  - mkdir travis-phantomjs
-  - wget https://github.com/JordiPolo/phantomjs/blob/master/phantomjs-2.1.1-linux-x86_64.tar.bz2?raw=true -O $PWD/travis-phantomjs/phantomjs-2.1.1-linux-x86_64.tar.bz2
-  - tar -xvf $PWD/travis-phantomjs/phantomjs-2.1.1-linux-x86_64.tar.bz2 -C $PWD/travis-phantomjs
-  - export PATH=$PWD/travis-phantomjs/phantomjs-2.1.1-linux-x86_64/bin:$PATH
+- mkdir travis-phantomjs
```
Comment (jfeltesse-mdsol) [inline .travis.yml:10]: why the indent change? it makes the file much harder to read

Comment (jcarres-mdsol) [reply .travis.yml:10]: The Travis tool sets up the encrypted key and formats the file this way

**Comment by ykitamura-mdsol** [comment]
looks good to me

**Comment by jfeltesse-mdsol** [comment]
seems good to go, one more thing though is, do we need to keep ruby 2.1 in the matrix? and also the other versions are a bit dated.

**Pull request merged**